cmd/messaging-tool/send.go

```diff
@@ -75,15 +75,16 @@ func runSend(cmd *cobra.Command, args []string) {
 		return
 	}
 
-	c = &stomp.Connection{
+	// Set the clients variables before we can open it.
+	c, _ = stomp.NewConnection(&stomp.ConnectionBuilder{
```

Comment:
I know we currently don't return an error but if this is a usage example we should check and handle possible errors

Reply:
:+1: Done